make.go

```diff
@@ -236,12 +236,16 @@ func createGitRepository(debsrc, gopkg, orig string) (string, error) {
 		return dir, err
 	}
 
-	if err := runGitCommandIn(dir, "config", "user.name", getDebianName()); err != nil {
-		return dir, err
+	if debianName := getDebianName(); debianName == "TODO" {
```

Comment:
Then I still might not have understood the problem correctly. Can you add a `log.Printf("debian name = %q, debian email = %q\n", getDebianName(), getDebianEmail())` and let me know the output please?